docs/workflow/intern-onboarding.md

```diff
@@ -0,0 +1,20 @@
+<h3>Join Microsoft / .Net Foundation organizations</h3>
+
+1. Link Microsoft account with the Github account and join Microsoft GitHub organization:
+    + Go to https://opensourcehub.microsoft.com/link/ and https://repos.opensource.microsoft.com/ to link.
+    + Go to https://repos.opensource.microsoft.com/Microsoft to join Microsoft org.
+    + Go to  https://repos.opensource.microsoft.com/DotNet to join DotNet org.
+    + Join this group as well https://repos.opensource.microsoft.com/dotnet/teams/mlnet if you want to do PR via codeflow.
```

Comment:
Nit: What does ML.NET has to do with codeflow?

Reply:
I merged the peaces of guidelines I got by interacting with other members for any new contributors (it took me a week to get /azp command privileges). Giving your previous comment, I think this file should only link that document and tells one to ask for the /azp commands privileges.